setup.py

```diff
@@ -89,7 +89,8 @@
     "einops",
     "timm",
     "torchvision",
-    "scikit-image"
+    "scikit-image",
+    "mediapipe",
```

Comment:
I'm a bit worried about the protobuf dependency here: https://github.com/google/mediapipe/blob/a6c35e9ba5b7b877b8d907d0bf5b5c09d8dab6f3/requirements.txt#L7 => this could mess up other installs

Reply:
Then it would be better user to install mediapipe by their own. I will remove this.

Reply:
Thanks!